platform/mellanox/mlnx-platform-api/sonic_platform/smartswitch_thermal_updater.py

```diff
@@ -82,14 +82,6 @@ def start(self):
         self.load_tc_config_dpu()
         if self.host_mgmt_mode:
```

Comment:
you don't need to check this, please test on bobcat

Reply:
Removed. Test also passed in bobcat with FW and SW control mode.